### DISCUSSION TITLE
Illogical behaviour in the single-type select

### DISCUSSION THREAD

**Discussion by gdzhv200** [orchidjs/tom-select#399] [Ideas]
When you choose the select type single, after selecting an item from the list it still offers me to type something in the input. It's illogical. Also when the length of the selected item is too long it breaks the design, because doesn't wrap and neither stays inline. We solved this situation with @OlegTar and now it looks fine. Consider including this in future updates, thank you.

**Comment by jclusso** [comment]
For those finding this, https://github.com/orchidjs/tom-select/discussions/563 solves it.
